### ISSUE TITLE
Problema com System embarcada e rGuard

### ISSUE THREAD

**Issue by AnibalDuarte** [danielbarion/JBlueHeart-Source#11]
Ao colocar a system disponibilizada para download no Google Drive ele alertou para uma infecção:

![image](https://user-images.githubusercontent.com/3143872/51355258-bc215700-1a9d-11e9-836e-2ce4b63f9245.png)

O scan do Malwarebytes denunciou a presença do PUP.Optional.eSupportUndeletePlus

**Comment by danielbarion** [comment]
A system é uma system de instalação limpa do highfive patched para funcionar sem o game guard e já apontada para 127.0.0.1;

Caso não ache a system confiável, instale o highfive e faça o patch na system; não se esqueça de adicionar os arquivos necessários para rodar os items customs listados no readme do repositório;

**Comment by AnibalDuarte** [comment]
Sobre a questão do "virus" na verdade é um falso positivo causado por um arquivo de cache do UndeletePlus.

Outra questão surgiu: ativei o rGuard e as verificações de HWID e quando tento selecionar o servidor sou desconectado. No console do servidor é exibida a mensagem: **INFO: HWID Status: NoPatch!!!**

**Comment by danielbarion** [comment]
Entendi;

A system disponível é a mesma que utilizamos quando estávamos desenvolvendo;

Como o projeto é open source, tudo que utilizamos, colocamos para download junto com a source;

**Comment by AnibalDuarte** [comment]
Certo, mas esta system não tem a proteção do rGuard configurada, certo? Estou usando ela e ele não funciona, nesse sentido que me referi.


De: Daniel Barion <notifications@github.com>
Enviada em: sexta-feira, 18 de janeiro de 2019 14:40
Para: danielbarion/JBlueHeart-Source <JBlueHeart-Source@noreply.github.com>
Cc: Anibal Duarte <anibal.duarte@outlook.com>; Author <author@noreply.github.com>
Assunto: Re: [danielbarion/JBlueHeart-Source] Problema com System embarcada e rGuard (#11)


Entendi;

A system disponível é a mesma que utilizamos quando estávamos desenvolvendo;

Como o projeto é open source, tudo que utilizamos, colocamos para download junto com a source;

**Comment by danielbarion** [comment]
Exatamente, ao menos eu nunca usei algum nela...
